app/src/main/java/com/example/httpexample/MainActivity.kt

```diff
@@ -81,6 +135,12 @@ class MainActivity : AppCompatActivity() {
         }
         httpUrlConnection.responseCode
         httpUrlConnection.disconnect()
-        getBooksAndShowIt()
+        getBooksAndShowIt()*/
+    }
+}
+
+fun AppCompatActivity.toast(text: String) {
```

Comment:
This code should not be here. It's better to move it to some module with extensions